### ISSUE TITLE
Model training stalls with deepedit_multilabel

### ISSUE THREAD

**Issue by pwrightkcl** [Project-MONAI/MONAILabel#641]
**Describe the bug**
On starting model training from Slicer, training proceeds in the server output, but after a few minutes stops. The log file attached was taken after about 3 hours of inactivity after the last time stamp.
[app.log](https://github.com/Project-MONAI/MONAILabel/files/8100778/app.log)

**To Reproduce**
Steps to reproduce the behaviour:
1. See full details at https://github.com/Project-MONAI/MONAILabel/discussions/640.
2. Install monailabel-weekly.
3. `monailabel apps --download --name deepedit_multilabel --output apps`
4. Edit deepedit_multilabel/main.py to change segmentation labels to:
```
        self.label_names = {
            "haemorrhage": 1,
            "calcification": 2,
            "background": 0,
        }
```
5. Place images and labels in dataset directory as per MONAIlabel requirements.
    * Images in main directory
    * Ground truth labels in ./labels/final with same filenames as base images
6. Install Slicer 4.13.0-2022-02-15.
7. Install MONAIlabel plugin.
8. Start server without using pretrained model:
```
monailabel start_server \
  --app apps/deepedit_multilabel \
  --studies my_dataset \
  --conf use_pretrained_model false \
  --conf auto_update_scoring false \
  --conf heuristic_planner false \
  --conf tta_enabled false \
  --conf tta_samples 10
```
9. In Slicer, go to the MONAIlabel module.
10. In the 'Reload & Test' pane, click the reload icon to connect to the server.
11. In the 'Active Learning' pane, click 'Run'

**Expected behaviour**
The server output should run through the iterations and epochs until complete. Instead, it stops after a few epochs (out of 50).

**Screenshots**
I have attached a log, which should be more informative.

**Environment**
Ubuntu 20.04. MONAI in a fresh virtual environment based on Python 3.8.
Ensuring you use the relevant python executable, please paste the output of:

```
python -c 'import monai; monai.config.print_debug_info()'
```

Attached: 
[monai_config.txt](https://github.com/Project-MONAI/MONAILabel/files/8100807/monai_config.txt)

**Additional context**
The images I am using are CT brains with haemorrhage. The labels demarcate the lesion and regions of calcification. I have 52 images. I removed the labels from ten of them to test out active learning based on a model trained on the other 42 images.

When the training stalls, the GPU usage drops to minimal, but the CPU usage by Python sticks at 100%. (As an aside, I am wondering why it is capped at a single thread).

I have started training with 'Train:' in the 'Active Learning' pane set both to the default of 'ALL' and to 'deepedit_train' and both stall as described.

**Comment by diazandr3s** [comment]
Hi @pwrightkcl,

Thanks for the detailed explanation.
This is indeed one issue we've faced before. Yesterday we merged a PR solving it. 

The problem was this transform ``AddRandomGuidanceCustomd``. It enters in an infinite loop when the number of clicks were not generated https://github.com/Project-MONAI/MONAILabel/blob/2ff9bbe10bf5627193b6695dcd63b25be2b9b973/monailabel/deepedit/multilabel/transforms.py#L630

You should be able to train the deepedit_multilabel with the new monailabel-weekly released this Monday or directly from the GitHub repo.

Thanks again and please let us know how this goes :)

**Comment by diazandr3s** [comment]
Closing issue
